### PR TITLE
Simulate scheduler with debuggable scheduler container

### PR DESCRIPTION
**Pull request by YamasouA**
#### Which issue(s) this PR fixes:

Fixes #

#### Special notes for your reviewer:

/label tide/merge-method-squash

### CODE REVIEW

File: simulator/docs/sample/debuggable-scheduler/scheduler.yaml [hunk 1]
```diff
@@ -1,11 +1,4 @@
 kind: KubeSchedulerConfiguration
 apiVersion: kubescheduler.config.k8s.io/v1
 clientConnection:
-  kubeconfig: kubeconfig.yaml
```
Comment (sanposhiho) [inline simulator/docs/sample/debuggable-scheduler/scheduler.yaml:4]: why is it removed?

File: simulator/docs/sample/debuggable-scheduler/kubeconfig.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ kind: Config
 
 clusters:
   - cluster:
-      server: http://localhost:3131
```
Comment (sanposhiho) [inline simulator/docs/sample/debuggable-scheduler/kubeconfig.yaml:6]: ditto, why do we have to update it?

**Review by sanposhiho — COMMENTED** [review]
can you update the extender.md like we discussed?
https://github.com/kubernetes-sigs/kube-scheduler-simulator/pull/358#pullrequestreview-2160600457

File: simulator/docs/sample/debuggable-scheduler/scheduler.yaml (hunk 1 above)
Comment (YamasouA) [reply simulator/docs/sample/debuggable-scheduler/scheduler.yaml]: I fix this diff.
[da41f80](https://github.com/kubernetes-sigs/kube-scheduler-simulator/pull/353/commits/da41f800cee7789337bacfc3cfec1abd33713681)

**Pull request closed without merging**